docs/welcome-story.mdx

```diff
@@ -124,15 +124,16 @@ If you just want to try our components for demnstrations and so on, you can use
 ### JavaScript framework integration
 
 In addition to the available Web Component versions of Carbon components, this
-library also includes framework wrappers for Angular, React, and Vue if the
-desire is to use instead of the pure framework versions of Carbon components.
+library also supports usage with JavaScript frameworks such as Angular, React,
+and Vue if the desire is to use instead of the pure framework versions of
+Carbon components.
```

Comment:
```suggestion
library also supports usage with JavaScript frameworks like Angular, React,
and Vue if the desire is to use instead of the pure framework versions of
Carbon components. Specifically for React, this library comes with a wrapper
implementation around the Carbon Web Components for more seamless integration
with your React application.

```